src/System.Security.Cryptography.Xml/tests/TransformTest.cs

```diff
@@ -57,11 +61,47 @@ public override Type[] OutputTypes
 
     public class TransformTest
     {
+        [Fact]
+        public void Constructor()
+        {
+            ConcreteTransform concreteTransform = new ConcreteTransform();
+            Assert.Null(concreteTransform.Context);
+            Assert.Null(concreteTransform.Algorithm);
+#if !NET45 && !NET451 && !NET452 && !NET46 && !NET461 && !NET462 && !NET463
+            Assert.Empty(concreteTransform.PropagatedNamespaces);
+#endif
+        }
 
         [Fact]
         public void GetDigestedOutput_Null()
         {
             Assert.Throws<NullReferenceException>(() => new ConcreteTransform().GetDigestedOutput(null));
         }
+
+#if !NET45 && !NET451 && !NET452 && !NET46 && !NET461 && !NET462 && !NET463
+        [Theory]
+        [InlineData(typeof(XmlDsigC14NTransform))]
+        [InlineData(typeof(XmlDsigExcC14NTransform))]
+        [InlineData(typeof(XmlDsigC14NWithCommentsTransform))]
+        public void PropagatedNamespaces(Type type)
+        {
+            XmlDocument doc = new XmlDocument();
+            doc.AppendChild(doc.CreateElement("foo", "urn:foo"));
+            doc.DocumentElement.AppendChild(doc.CreateElement("bar", "urn:bar"));
+            Assert.Equal(string.Empty, doc.DocumentElement.GetAttribute("xmlns:f"));
+            Transform transform = type.GetConstructor(Type.EmptyTypes).Invoke(new object[0]) as Transform;
```

Comment:
I selected a list of transforms that did not actually change the XML to get as broad a coverage as possible. I omitted those that changed the output. I added a comment to clarify the selection of test data. I hope that answers your question, @krwq .